cobigen/cobigen-core-parent/cobigen-core-api/src/main/java/com/devonfw/cobigen/api/constants/ConfigurationConstants.java

```diff
@@ -19,5 +19,6 @@
     /** Filename of the {@link Properties} used to customize cobigen properties and template relocation. */
     public static final String COBIGEN_PROPERTIES = "cobigen.properties";
 
-    public static final String TEMPLATE_RESOURCE_FOLDER = "src/main/templates";
+    /** Source folder containing templates */
+    public static final String TEMPLATE_SOURCE_FOLDER = "src/main/templates";
```

Comment:
It's a resource folder ;) Source folders are just folder, which compilable resources.

Reply:
I thought that templates are technically compiled, that's why i changed it

Reply:
No, this notion of source and resource is coming from maven standard. It relates to the compile step in maven, which does not effect the template in any sense.